packages/components/nav/src/item_wrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+
+import { Flex, FlexProps } from "@buoysoftware/anchor-layout";
+
+interface OwnProps {
+  active?: boolean;
+}
+
+export type ItemWrapperProps = OwnProps & FlexProps;
+
+export const ItemWrapper = styled(({ active = false, ...props }) => {
+  return (
+    <Flex
+      alignItems="center"
+      borderBottomStyle="solid"
+      borderBottomWidth="3px"
+      borderBottomColor={active ? "interactive.default" : "transparent"}
+      display="flex"
+      height="100%"
+      mx="xxl"
+      py="l"
+      as="li"
+      {...props}
+    />
+  );
+})<ItemWrapperProps>`
+  text-decoration: none;
+  transition: opacity 0.1s ease-in-out, border-color 0.3s ease-in-out;
+
```

Comment:
This was extracted out from the `Nav` component. However, I ended up removing the following CSS

```
a {
    text-decoration: none;
    color: inherit;
  }
```

because this was overwriting the `text-decoration` and `color` properties from the link components themselves. IMO this was probably just here to make the storybook story match the design. In actual implementation, the link components themselves should declare their own styles